sabnzbd/cfg.py

```diff
@@ -92,6 +92,7 @@ def validate_server(value):
 enable_filejoin = OptionBool('misc', 'enable_filejoin', True)
 enable_tsjoin = OptionBool('misc', 'enable_tsjoin', True)
 enable_par_cleanup = OptionBool('misc', 'enable_par_cleanup', True)
+prospective_par_download = OptionBool('misc', 'prospective_par_download', True)
```

Comment:
It was optional because I didn't know if everyone would like it by default, since there might be some overhead. But my thought was also to enable it by default. 
Don't know what would be a sensible combo with download all par, so will change it to enable by default :)